src/react/src/util/propTypes.js

```diff
@@ -377,14 +377,17 @@ export const facilityClaimPropType = shape({
     contact_person: string.isRequired,
     job_title: string.isRequired,
     email: string.isRequired,
-    phone_number: string.isRequired,
     company_name: string.isRequired,
     website: string.isRequired,
     facility_description: string.isRequired,
     linkedin_profile: string.isRequired,
     status: oneOf(Object.values(facilityClaimStatusChoicesEnum)).isRequired,
     contributor: shape({}).isRequired,
     facility: facilityPropType.isRequired,
+    facility_website: string.isRequired,
```

Comment:
I'm not sure whether we still need to use `facility_...` prefix since we are going to migrate to `production-location`.

Reply:
These data come from the back-end and are stored in the database with the `facility_...` prefix.
I believe renaming these fields should be within the scope of another separate ticket for these purposes.

Reply:
Ack.